oneclick.py

```diff
@@ -682,7 +667,7 @@ def isbn_info_to_metadata(cls, book, include_bibliographic=True, include_formats
                     subjects.append(subject)
 
             # audience options are: adult, beginning-reader, childrens, young-adult
-            # NOTE: audience can be set to "Adult" while publisher is "HarperTeen".
+            # NOTE: In OneClick metadata, audience can be set to "Adult" while publisher is "HarperTeen".
```

Comment:
I filed https://github.com/NYPL-Simplified/server_core/issues/426 to address this.